src/simplay-jupyter/src/accurateSlider.ts

```diff
@@ -1,5 +1,5 @@
 export class AccurateSlider {
-  private static PADDING = 2;
+  private static PADDING = 0;
```

Comment:
```suggestion
  private static PADDING = 2;
```

Please leave some padding, as this is needed so that scrolling to 0 or MAX is possible. The alternative solution is to add padding to the canvas of the simulation, so that they align.